web/crux/proto/crux.proto

```diff
@@ -629,8 +629,10 @@ message ContainerStatusItem {
   string name = 101;
   string command = 103;
   google.protobuf.Timestamp createdAt = 104;
-  ContainerStatus status = 105;
-  string state = 106;
+  /** The 'State' of the container (Created, Running, etc) */
+  ContainerState state = 105;
+  /** The 'Status' of the container ("Created 1min ago", "Exited with code 123", etc) */
+  string status = 106;
```

Comment:
Same here.

Reply:
These are copies from the `protobuf` folder.